### DISCUSSION TITLE
Overlay permission

### DISCUSSION THREAD

**Discussion by thproflord** [dofire/Floating-Bubble-View#67] [General]
While testing the library implementation before deploying to production, we realized that one phone, the Samsung A03Core, does not have the permission to display on top. This is because it is a low-end device. We are gathering information to see how much this would affect us. We believe it could be the Android Go operating system, but we are not sure. Do you know anything more about this?

**Comment by dofire** [comment]
@thproflord This is the first time I've heard about this, so unfortunately I'm unable to help. I'll try to research about this issue if I have time. Thanks
